example/build.gradle

```diff
@@ -2,6 +2,12 @@ plugins {
     id 'com.android.application'
 }
 
+def localProperties = new Properties()
+def localPropertiesFile = rootProject.file('local.properties')
+if (localPropertiesFile.exists()) {
+    localProperties.load(new FileInputStream(localPropertiesFile))
+}
```

Comment:
Leverage `.gitignore`ed `local.properties` for secure placing of secrets during local development and testing